{
  "modified": "2025-10-23T19:17:26Z",
  "published": "2025-10-23T19:17:26Z",
  "schema_version": "1.5.0",
  "id": "MAL-2025-48911",
  "summary": "Malicious code in yuzo (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "yuzo",
        "purl": "pkg:pypi/yuzo"
      },
      "versions": [
        "1.0.0",
        "1.0.1",
        "1.0.2",
        "1.4.3"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2025-05231",
        "import_time": "2025-10-27T18:08:51.200751338Z",
        "modified_time": "2025-10-23T19:17:26Z",
        "sha256": "9aca61cad4edb45dc2317dc31bd953db265a08c9ce0668699777692b62339b71",
        "source": "reversing-labs",
        "versions": [
          "1.0.0",
          "1.0.1",
          "1.0.2",
          "1.4.3"
        ]
      }
    ]
  }
}
